chain/bitcoind_client.go

```diff
@@ -429,22 +462,14 @@ func (c *BitcoindClient) Start() error {
 			"%v", err)
 	}
 
-	c.bestBlockMtx.Lock()
 	c.bestBlock = waddrmgr.BlockStamp{
```

Comment:
sane mutex question

Reply:
I don't think it's necessary here since this is the `Start` method, but added anyway.